core/src/main/scala/org/apache/spark/deploy/worker/StandaloneWorkerShuffleService.scala

```diff
@@ -31,7 +31,6 @@ import org.apache.spark.network.shuffle.ExternalShuffleBlockHandler
  *
  * Optionally requires SASL authentication in order to read. See [[SecurityManager]].
  */
-private[worker]
 class StandaloneWorkerShuffleService(sparkConf: SparkConf, securityManager: SecurityManager)
```

Comment:
The correct thing to do here is not to remove its visibility, but to abstract this class so it is not used exclusively for standalone mode.